### DISCUSSION TITLE
Beep after scan

### DISCUSSION THREAD

**Discussion by jakubmilata** [mebjas/html5-qrcode#295] [General]
I am trying to beep after scan. 
I use simple function:
function beep() {
sound.play();
}
(sound is defined out of this function)
It the function works fine when I call it from click of a button.. 
But it does not want to work in the onScanSuccess function. 

I have heard a very short beep a few times after scan. It seems that the function actually works but it is interrupted before the whole sound is played. Is here anyone able to help?

**Comment by ROBERT-MCDOWELL** [comment]
you have certainly an issue with your browser or javascript

**Reply by jakubmilata** [reply]
I am tesing it in mobile chrome on iphone. 
Function beep(); works perfectly in that browser, when it is called from click of a button..

When I add whatever to function beep, like console_log, or alert or something, that function is running after scan as it should. It just does not play the whole beep. Only sometimes maybe 0.1s or a very very short beep.

**Reply by jakubmilata** [reply]
I just found out that it works great in chrome on samsung phone. The problem is on iphone only..

**Comment by ROBERT-MCDOWELL** [comment]
So you must contact safari(?) developers, not here. please close the issue.
